### PR TITLE
KEYCLOAK-11196 Document the `useNonce` option for JS adapter

### PR DESCRIPTION
**Pull request by jonkoops**
See: https://github.com/keycloak/keycloak/pull/6272

### CODE REVIEW

**Comment by jonkoops** [comment]
Looks like the external link check is failing, but it seems unrelated to this PR.

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 1]
```diff
@@ -302,6 +302,7 @@ Called to initialize the adapter.
 
 Options is an Object, where:
 
+* useNonce - Adds a https://en.wikipedia.org/wiki/Cryptographic_nonce[cryptographic nonce] to verify that the authentication response matches the request (default is true).
```
Comment (stianst) [inline securing_apps/topics/oidc/javascript-adapter.adoc]: Can you remove the link please and just write cryptographic nonce?

**Review by stianst — CHANGES_REQUESTED** [review]

File: securing_apps/topics/oidc/javascript-adapter.adoc (hunk 1 above)
Comment (jonkoops) [reply securing_apps/topics/oidc/javascript-adapter.adoc]: Done :+1:

**Review by stianst — APPROVED** [review]

**Comment by stianst** [comment]
Thanks

**Pull request merged**